builder/site.hs

```diff
@@ -1,15 +1,39 @@
 --------------------------------------------------------------------------------
+{-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE OverloadedStrings #-}
-import           Data.Monoid           ((<>))
+import           Control.Monad
+import           Data.Aeson
+import qualified Data.ByteString.Lazy          as BL
+import           Data.Monoid                   ((<>))
 import           Data.Time.Calendar
 import           Data.Time.Clock
 import           Hakyll
+import           Hakyll.Core.Compiler
+import           Hakyll.Core.Compiler.Internal
+import           Hakyll.Core.Provider
+import           Hakyll.Core.Routes
 import           System.FilePath.Posix
+import           Testimonial
 
 
 --------------------------------------------------------------------------------
 main :: IO ()
 main = mkContext >>= \ctx -> hakyll $ do
+  match "testimonials/logos/*" $ do
+    route idRoute
+    compile copyFileCompiler
+
+  match "testimonials/*.yaml" $ do
+    compile parseTestimonialCompiler
+
+  create ["testimonials.json"] $ do
+    route idRoute
+    compile $ do
+      testimonials <- loadAll @Testimonial "testimonials/*.yaml"
+      item <- makeItem $ (BL.unpack . encode . map itemBody) testimonials
```

Comment:
I like the use of `(.)`.  Why not go all the way and use `(makeItem . BL.unpack . encode . map itemBody) testimonials`?

Reply:
ah good call, no idea why I didn't do that in the first place TBH